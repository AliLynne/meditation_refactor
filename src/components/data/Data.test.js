import React from 'react'
import { shallow } from 'enzyme'

import Data from './Data'

describe('Data component', () => {
  it('should render correctly', () => {
    const data = shallow(<Data />)

    expect(data.debug()).toMatchSnapshot()
  })
})
