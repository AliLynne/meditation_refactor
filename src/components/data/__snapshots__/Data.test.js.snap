// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Data component should render correctly 1`] = `"<div />"`;
